app/DynamicsAdapter/Fams3Adapter.Dynamics/SearchRequest/SearchRequestService.cs

```diff
@@ -141,5 +145,21 @@ public async Task<SSG_Asset_WorkSafeBcClaim> CreateCompensationClaim(SSG_Asset_W
         {
             return await this._oDataClient.For<SSG_Asset_WorkSafeBcClaim>().Set(claim).InsertEntryAsync(cancellationToken);
         }
+
+        public async Task<SSG_Asset_ICBCClaim> CreateInsuranceClaim(ICBCClaimEntity claim, CancellationToken cancellationToken)
+        {
+            return await this._oDataClient.For<SSG_Asset_ICBCClaim>().Set(claim).InsertEntryAsync(cancellationToken);
+        }
+
+        public async Task<SSG_SimplePhoneNumber> CreateSimplePhoneNumber(SSG_SimplePhoneNumber assetPhone, CancellationToken cancellationToken)
```

Comment:
it may not be asset related phones only, can we change parameter 'assetPhone' to simplePhone ?

Reply:
Oh we can just use phone?

Reply:
sure